core/src/main/java/org/kohsuke/stapler/interceptor/Interceptor.java

```diff
@@ -45,6 +46,8 @@ public void setTarget(Function target) {
      * @return
      *      Return value from the method.
      * @throws InvocationTargetException if you want to send e.g. something from {@link HttpResponses}
+     * @throws CancelRequestHandlingException
```

Comment:
`CancelRequestHandlingException` as stated in javadoc. Or maybe I'm missing your point.